src/main/java/se/sundsvall/webmessagecollector/integration/opene/OpenEClient.java

```diff
@@ -1,5 +1,9 @@
 package se.sundsvall.webmessagecollector.integration.opene;
```

Comment:
out of scope but it is possible to replace this with feign

Reply:
I dont see why it shouldn't be possible - but as you say, out of scope for this task (which already has some scope creep 🙂).